### ISSUE TITLE
JS implementation to write extra markup structure for IE only

### ISSUE THREAD

**Issue by alicetseng** [ucla-oarc-web/WebBlocks#127]
Refs #31
Twitter Bootstrap uses a navbar-inner wrapper div as a work-around to prevent IE from breaking when gradients+overflow are clashing on the same entity. This is a known/documented work-around. WebBlocks has chosen to take a JS + IE Only stylesheet to address this. Will need this implementation, otherwise navbar will break in all IE.

**Comment by ebollens** [comment]
@atsengucla please provide the prototype without this, and then just in this ticket give me the markup that I need to add and the IE CSS needed. This will be easy enough to add in via Javascript to fix IE.

**Comment by alicetseng** [comment]
sure. will upload it to the same template (navbar) in an IE sub-directory.

**Comment by alicetseng** [comment]
@ebollens OK. IE only code has been uploaded to the template folder. Basically, there needs to be an extra child div (which I named chrome - since it just contains the gradient and outline effects of the bar). Tested fine on IE and things are looking OK. IE 8 will break in the element with the footer but that's due to IE 8 not understanding the html5 entity footer. so won't be shimmed here since it'll be taken care of on the global level.
